docs/configuration.md

```diff
@@ -0,0 +1,235 @@
+
+# Migrate existing services to nix-bitcoin
+
+## Example: bitcoind
+
+```shell
+# 1. Stop bitcoind on your nodes
+ssh root@nix-bitcoin-node 'systemctl stop bitcoind'
+# Also stop bitcoind on the node that you'll be copying data from
+
+# 2. Copy the data to the nix-bitcoin node
+# Important: Add a trailing slash to the source path
+rsync /path/to/existing/bitcoind-datadir/ root@nix-bitcoin-node:/var/lib/bitcoind
+
+# 3. Fix data dir permissions on the nix-bitcoin node
+ssh root@nix-bitcoin-node 'chown -R bitcoin: /var/lib/bitcoind'
+
+# 4. Start bitcoind
+ssh root@nix-bitcoin-node 'systemctl start bitcoind'
+```
+
+You can use the same workflow for other services.\
+The default data dir path is `/var/lib/<service>` for all services.\
+Some services require extra steps:
+
+- lnd
+
+  Copy your wallet password to `$secretsDir/lnd-wallet-password` (See: [Secrets dir](#secrets-dir)).
+
+- btcpayserver
+
+  Copy the postgresql database:
+  ```shell
+  # Export (on the other node)
+  sudo -u postgres pg_dump YOUR_BTCPAYSERVER_DB > export.sql
+  # Restore (on the nix-bitcoin node)
+  sudo -u postgres psql btcpaydb < export.sql
+  ```
+
+- joinmarket
+
+  Copy your wallet to `/var/lib/joinmarket/wallets/wallet.jmdat`.\
+  Write your wallet password, without a trailing newline, to
+  `$secretsDir/lnd-wallet-password` (See: [Secrets dir](#secrets-dir)).
+
+# Use bitcoind from another node
+
+Use a bitcoind instance running on another node within a nix-bitcoin config.
+
+```nix
+services.bitcoind = {
+  # Address of the other node
+  address = "10.10.0.2";
+  rpc.users = let
+    # The fully privileged bitcoind RPC username of the other node
+    name = "myrpcuser";
+  in {
+    privileged.name = name;
+    public.name = name;
+    ## Set this if you use btcpayserver
+    # btcpayserver.name = name;
+    ## Set this if you use joinmarket-ob-watcher
+    # joinmarket-ob-watcher.name = name;
+  };
+};
+# Disable the local bitcoind service
+systemd.services.bitcoind.wantedBy = mkForce [];
+```
+
+Now save the password of the RPC user to the following files on your nix-bitcoin node:
+```shell
+$secretsDir/bitcoin-rpcpassword-privileged
+$secretsDir/bitcoin-rpcpassword-public
+
+## Only needed when set in the above config snippet
+# $secretsDir/bitcoin-rpcpassword-btcpayserver
+# $secretsDir/bitcoin-rpcpassword-joinmarket-ob-watcher
+```
+See: [Secrets dir](#secrets-dir)
+
+# Temporarily disable a service
+
+Sometimes you might want to disable a service without removing the service user and
+integration with other services, as it would happen when setting
+`services.<service>.enable = false`.
+
+Use the following approach:
+```
+systemd.services.<service>.wantedBy = mkForce [];
+```
+This way, the systemd service still exists, but is not automatically started.
+
+# Appendix
+
+## Secrets dir
+
+The secrets dir is set by option `nix-bitoin.secretsDir` and has the
+following default values:
+- `/secrets` (if you're using the `secure-node.nix` template)
```

Comment:
Aren't passwords kept in `/var/src/secrets`?

Reply:
Ah, right, when krops is used. Fixed.